src/index.html

```diff
@@ -1,14 +1,92 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="menu">
+          <li class="list-item">
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="list__item">
```
страйся дотримуватись БЕМ-у
https://ru.bem.info/methodology/